toolchain/check/import_cpp.cpp

```diff
@@ -193,8 +194,8 @@ static auto GenerateAst(Context& context, llvm::StringRef importing_file_path,
       GenerateCppIncludesHeaderCode(context, imports),
       // Parse C++ (and not C).
       {"-x", "c++",
-       // Propagate the target to Clang.
-       "-target", target.str()},
+       // Propagate compile flag to Clang.
+       "-target", target.str(), use_pie ? "-fpie" : "-fno-pie"},
```

Comment:
We should probably use `-fPIE` instead of `-fpie` for robustness for larger binaries.

Reply:
Done, didn't realize capitalization was significant